src/components/Report.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Parallax } from "react-parallax";
+
+class Report extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      img: "https://res.cloudinary.com/hg7jltnn9/image/upload/f_auto,q_auto/v1562031427/public/assets/img/pan-with-olive-oil-ready-to-cooking-picjumbo-com_1024_x683_r6v9jd.jpg"
+    };
+  }
+
+  render() {
+    return (
+      <div>
+        <div className="hide-on-small-only">
+          <Parallax bgImage={this.state.img} strength={500}>
```

Comment:
Does <Report/> need to be a stateful component? It uses state only once, and it's for storing the parallax image.